python/src/BetaFactory_doc.i.in

```diff
@@ -5,14 +5,14 @@ Available constructor:
     BetaFactory()
 
 Let :math:`n` be the sample sample size. 
-Let :math:`x_{min}` be the sample minimum and :math:`x_{max}` be the sample maximum. 
+Let :math:`x_{(1)}` be the sample minimum and :math:`x_{(n)}` be the sample maximum. 
 Let :math:`\delta` be the sample range:
 
 .. math::
     :nowrap:
 
     \begin{eqnarray*}
```

Comment:
Are the `eqnarray*` environment and the :nowrap: command really useful?

Reply:
I do not known... I did this online. Testing this would require a build.

Reply:
Got it: nowrap means that the equation is printed on a single line. Otherwise, I suppose that an equation could be split on several lines if it is too long. The eqnarray environment is not necessary for the first equation, but it is for the remaining ones.

Reply:
Fixed.